### PR TITLE
Update VEP to release 101 (August 2020).

### PR DESCRIPTION
**Pull request by mbookman**
I used these changes to build a new Docker image and cache file.
I then ran it through the [gcp-variant-transforms annotation pipeline](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/docs/variant_annotation.md).
All seemed to work fine.

The changes here can be used to update the defaults for gcp-variant-transforms as described in this issue:

- [gcp-variant-transforms/issues/665](https://github.com/googlegenomics/gcp-variant-transforms/issues/665): Update the VEP release used for annotations

### CODE REVIEW

File: batch/vep/build_vep_cache.sh [hunk 1]
```diff
@@ -81,7 +81,11 @@ else
   curl -O "${remote_fasta}.gzi"
 fi
 
-readonly remote_cache="${ftp_base}/variation/VEP/${cache_file}"
+if (( release <= 95 )); then
```
Comment (samanvp) [inline batch/vep/build_vep_cache.sh:85]: Up in this file we have a variable default to version 91, could you please update that. Also in the comments above we mention default is 91, let's update that as well.

**Review by samanvp — APPROVED** [review]
Could you please sync to #12  and also address the single comment I added.

Thank you.

File: batch/vep/build_vep_cache.sh (hunk 1 above)
Comment (mbookman) [reply batch/vep/build_vep_cache.sh:85]: updated. PTAL.

Comment (mbookman) [reply batch/vep/build_vep_cache.sh:85]: Missed updating to #12. Will do that now.

**Pull request merged**